app/controllers/characters_controller.rb

```diff
@@ -31,14 +37,26 @@ def update
       render :edit
     end
   end
-  
+
   def show; end
 
   def destroy
     @character.destroy
     redirect_to characters_path, flash: { notice: "Character has been destroyed" }
   end
 
+  def increment_rank
+    @character = Character.find(params[:character_id])
+    @skill = @character.skills.find_by(name: params[:skill_name])
+    @character.verify_skill(@skill, "increment")
```

Comment:
I don't feel like verify_skill is a good method name, also I don't like the "increment", "decrement" string